fastly.toml

```diff
@@ -5,7 +5,7 @@ authors = ["zeeshan.ashraf@optimizely.com"]
 description = "A basic starter kit for Fastly's Compute@Edge with Optimizely built in."
 language = "javascript"
 manifest_version = 2
-name = "optimizely"
+name = "Optimizely"
```

Comment:
@triblondon - what does the capitalization of "optimizely" achieve here? Does this change how the starter kit is executed behind the scenes on Fastly?

Reply:
@triblondon – bump

Reply:
Tagging @hrmsk66 & @Integralist, as you both seem to maintain [the Fastly JS Starter Kit](https://github.com/fastly/compute-starter-kit-javascript-default). Maybe you can answer this?

Reply:
Hi there, sorry for the delay, this is just so that Optimizely appears in a way that's consistent with our other starter kits.  If your brand style is to present the name in all lower case then we can keep this as is.  here's how it looks right now:

<img width="1173" alt="image" src="https://user-images.githubusercontent.com/1735391/184907806-5380b6be-6c21-4de3-8d7f-828081dc0442.png">

Reply:
Got it - thanks for the reply, makes sense. Uppercase "O" should be fine from a branding perspective!

cc: @griffincox-optimizely

Reply:
Gotcha. I think we should go with "Optimizely Full Stack" since these edge starter kits are for FS only. Thoughts @opti-jnguyen?